Add nethserver-hotsync-restore event for slave promotion to master

Stell0 opened the pull request:
- use nethserver-hotsync-restore instead of configuration restore
- nethserver-hotsync-restore restore configuration backup and data backup
- nethserver-backup-data is now required

gsanchietti commented on root/etc/e-smith/events/actions/nethserver-hotsync-slave-restore:
```diff
@@ -0,0 +1,24 @@
+#
+
+/sbin/e-smith/restore-config
+/sbin/e-smith/restore-data
```
This command will try to restore backup data from the remote source.

Since all data are already in place, you need to call only the post-restore-data event.

Stell0 replied:
👍

Stell0 committed:
Add nethserver-hotsync-restore event for slave promotion to master

- use nethserver-hotsync-restore instead of configuration restore
- nethserver-hotsync-restore restore configuration backup and data backup
- nethserver-backup-data is now required
